isis/tests/FunctionalTestsGetsn.cpp

```diff
@@ -0,0 +1,151 @@
+#include <iostream>
+#include <sstream>
+
+#include <QIODevice>
+#include <QTextStream>
+#include <QStringList>
+#include <QFile>
+
+#include "getsn.h"
+#include "Fixtures.h"
+#include "Pvl.h"
+#include "PvlGroup.h"
+#include "TestUtilities.h"
+#include "Cube.h"
+
+#include "gmock/gmock.h"
+
+using namespace Isis;
+
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/getsn.xml").expanded();
+static QString FROM_PARAM = "FROM=data/defaultImage/defaultCube.pvl";
```

Comment:
This path should be available from the DefaultCube fixture from either the cube or the pvl label, more likely from the cube. If we get the path from that it will reduce other instances of changing paths like this if the data area ever ends up changing.